src/lib/fragments/AgGrid.react.js

```diff
@@ -176,16 +176,17 @@ export default class DashAgGrid extends Component {
                 markdown: this.generateRenderer(MarkdownRenderer),
                 ...newComponents,
             },
-            pauseSelections: false,
             rerender: 0,
             openGroups: {},
             gridApi: null,
             columnState_push: true,
         };
 
         this.selectionEventFired = false;
+        this.pauseSelections = false;
         this.reference = React.createRef();
         this.pendingChanges = null;
+        this.dataUpdates = false;
```

Comment:
What are the new `pauseSelections`/`dataUpdates` props for?

Reply:
`pauseSelections` isnt new, its to keep systematic adjustments from triggering the `onSelectionChanged`, as far as `dataUpdates` its to be used in the event of `rowData` pushed from the server, this way it can ignore selection events. As this clears out the `selectedRows` if there is no data there:

`rowData` became a async process, so both were triggered at the same time. But because the `rowData` was updating, when the `selectedRows` were trying to be applied the grid was essentially empty.

Reply:
Got it. Just wondering, is there an open ticket that describes these issues? #274 looks like it's describing these issues..?

Reply:
Yes, this is linked with the two issues already. :)

Reply:
aha... yep catching up to you hah